sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -105,9 +106,11 @@ def submit(self, proto_pipeline):
     Submit and run the pipeline defined by `proto_pipeline`.
     """
     prepare_response = self.prepare(proto_pipeline)
+    artifact_endpoint = (self.artifact_endpoint
```

Comment:
Can we add a test for this to prevent future regressions?

Reply:
Sure thing I'll come up with something.